lib/plugins/create/templates/aws-golang/serverless.yaml

```diff
@@ -0,0 +1,51 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    v1.docs.serverless.com
+#
+# Happy Coding!
+
+service: aws-golang
```

Comment:
We've added a note here (you can copy it over from the other templates) that the user should update the services name.

Additionally we've added a newline below it so that it's easier to read.